packages/webview_flutter/lib/platform_interface.dart

```diff
@@ -31,6 +31,9 @@ abstract class WebViewPlatformCallbacksHandler {
 
   /// Report web resource loading error to the host application.
   void onWebResourceError(WebResourceError error);
+
+  /// Invoked by [WebViewPlatformController] when the URL has changed.
+  void onUpdateVisitedHistory(String url);
```

Comment:
The naming here seems very Android-centric; is there a reason not to call it something more generic like `onUrlChanged`?